CHARTER.md

```diff
@@ -46,21 +43,24 @@ This Technical Charter sets forth the responsibilities and procedures for techni
 
    - v. appointing representatives to work with other open source or open standards communities;
 
-   - vi. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
 
-   - vii. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the Technical Initiative; and
+ - f. The working group lead is responsible of
+
+   - i. facilitating discussions, seeking consensus, and where necessary, voting on technical matters relating to the Technical Initiative; and
 
-   - viii. coordinating any communications regarding the Technical Initiative.
+   - ii. coordinating any communications regarding the Technical Initiative.
+  
+   - iii. approving and implementing policies and processes for contributing (to be published in the Technical Initiative repository) and coordinating with the Linux Foundation to resolve matters or concerns that may arise as set forth in Section 6 of this Technical Charter;
 
-#### 3. TSC Voting
+#### 3. Voting
 
-- a. While the Technical Initiative aims to operate as a consensus-based community, if any TSC decision requires a vote to move the Technical Initiative forward, the voting members of the TSC will vote on a one vote per voting member basis.
+- a. While the Technical Initiative aims to operate as a consensus-based community, if any decision requires a vote to move the Technical Initiative forward, the working group will vote on a one vote per member basis.
```

Comment:
To be consistent, most of these instances of "working group" should probably be changed to "Technical Initiative."